incubator/hnc/pkg/config/default_config.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+import (
+	api "github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/api/v1alpha1"
+)
+
+// GetDefaultConfig creates the default configuration for HNCConfiguration Spec.
+// By default, HNC configuration should always propagate Roles and RoleBindings.
+// See details in http://bit.ly/hnc-type-configuration
+func GetDefaultConfig() *api.HNCConfigurationSpec {
```

Comment:
This is only returning the spec, so it should probably be called `GetDefaultConfigSpec` or something like that. Also, no need to return a pointer if the caller doesn't need it to be one (and I don't _think_ it does).

Reply:
Thanks for the suggestion. Changed the name and returned the object instead of pointer. You are right, it does not need to be a pointer.